ui/app/components/form-field-groups-loop.js

```diff
@@ -0,0 +1,3 @@
+import Component from '@glimmer/component';
+
+export default class FormFieldGroupsLoop extends Component {}
```

Comment:
Can we add JSDocs here so it's easily discoverable what the component takes as args?